src/intersection-observer-polyfill.js

```diff
@@ -233,12 +236,25 @@ export class IntersectionObserverPolyfill {
      * @private {Array<!ElementIntersectionStateDef>}
      */
     this.observeEntries_ = [];
+
+    /**
+     * Mutation observer to fire off on visibility changes
+     * @private {?Object}
+     */
+    this.mutationObserver_ = null;
```

Comment:
why is mutationobserver just an `Object` type?

Reply:
Done.